src/app/core/services/milestone.service.ts

```diff
@@ -45,6 +49,9 @@ export class MilestoneService {
 
     // add default milestone for untracked issues/PRs at the end
     milestoneData.push(Milestone.DefaultMilestone);
+    milestoneData.push(Milestone.IssueWithoutMilestone);
+    milestoneData.push(Milestone.PRWithoutMilestone);
+
```

Comment:
Since we are splitting having no milestones into issues and PRs, should we deprecate `Milestone.DefaultMilestone`?